features/verhuisd.intergemeentelijk.feature

```diff
@@ -0,0 +1,45 @@
+# language: nl
+Functionaliteit: gepubliceerde gebeurtenis 'verhuisd.intergemeentelijk' bij een aangifte van adreswijziging naar een andere gemeente
+  Als afdeling Begraven van de gemeente wil ik het weten wanneer een gevolgde persoon verhuisd is,
+  zodat ik batchgewijs automatische betalingen kan verwerken zonder het hele bestand in één keer te verversen.
+
+  Als afdeling Parkeerbelasting van de gemeente wil ik het weten wanneer een gevolgde persoon verhuisd is,
+  zodat ik de invordering naar het juiste adres kan sturen en het weet wanneer een persoon vertrokken is.
+ 
+  Als afdeling WGS Vroegsignalering van de gemeente wil ik het weten wanneer een gevolgde persoon verhuisd is naar een andere gemeente,
+  zodat ik het dossier kan overdragen aan de nieuwe gemeente en het dossier kan sluiten.
+
+  Achtergrond:
+    Gegeven het adres 'Burgemeester_Van_Der_Dussenplein_1_Hengelo'
+    * in gemeente 'Hengelo'
+    * met adresseerbaar object identificatie '0164010000047847'
```

Comment:
het ADO kan misschien weg, omdat dit niet nodig is om te illustreren dat persoon van de ene naar de andere gemeente verhuist.